mlem/contrib/requirements.py

```diff
@@ -0,0 +1,37 @@
+import logging
+import os
+from typing import ClassVar, Optional
+
+from mlem.core.objects import MlemBuilder, MlemModel
+from mlem.core.requirements import Requirements
+from mlem.ui import EMOJI_OK, EMOJI_PACK, echo
+
+REQUIREMENTS = "requirements.txt"
+
+logger = logging.getLogger(__name__)
+
+
+class RequirementsBuilder(MlemBuilder):
```

Comment:
What happens if requirements have something that should be installed with `apt-get`? Are you going to install it?

I assume we should, but on the other hand this may lead to re-installing and breaking (since `apt-get` / `brew` is not isolated). How should we handle this?

Reply:
Yes, this is a question that I am also not sure about. I think this might be going too far I guess. We cannot and we shouldn't be a tool to do every possible thing IMO. Maybe we shouldn't touch OS based dependencies. And it will only complicate stuff. I think `venv` and `conda` are at least good to start with for this PR though.

One more related angle is -- how to determine conda based packages automatically given a model. In the test I wrote, I added it manually for example.

Reply:
So, we don't install OS-based requirements with this? Let's add a short note to cmd description if possible, and cover this in docs. And print a warning after the command execution, if there are some OS deps for the model (if there are none, no reason to print this warning).

I would assume having something like `--os-deps` flag that will install them could be good. But, we can keep it a separate ticket for now.

Reply:
We can add support for this for unix only, so that you can use it to install in docker build for example. Probably it's better not to call `apt-get` directly, but instead allow to print requirements of specific type, so you can use it like this `apt-get install -y $(mlem build requirements -m model --type unix)`

Reply:
I have done the `unix` thingie and added a test for the same. But the argument is `platform` instead of `type` since `type` is already a `ClassVar`